python/lsst/ip/diffim/imageMapReduce.py

```diff
@@ -532,10 +535,12 @@ def _runMapper(self, exposure, doClone=False, **kwargs):
             raise ValueError('Bounding boxes list and expanded bounding boxes list are of different lengths')
 
         self.log.info("Processing %d sub-exposures", len(self.boxes0))
+        self.log.info("Mapper sub-task: %s", self.mapperSubtask._DefaultName)
```

Comment:
These subtasks should be written as part of the Config output: do we need them to be logged, also?